projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/b2b/tabbing-order.config.ts

```diff
@@ -0,0 +1,193 @@
+import { TabbingOrderConfig, TabbingOrderTypes } from '../tabbing-order.model';
+
+const paymentTypeRadio = {
+  value: 'paymentType',
+  type: TabbingOrderTypes.RADIO,
+};
+
+const backButton = { value: 'Back', type: TabbingOrderTypes.BUTTON };
+const continueButton = { value: 'Continue', type: TabbingOrderTypes.BUTTON };
+
+export const tabbingOrderConfig: TabbingOrderConfig = {
+  paymentMethod: [
+    {
+      value: 'poNumber',
+      type: TabbingOrderTypes.GENERIC_INPUT,
+    },
+    paymentTypeRadio,
+    paymentTypeRadio,
+    backButton,
+    continueButton,
+  ],
+  shippingAddressNew: [
+    { value: 'Method ofPayment', type: TabbingOrderTypes.LINK },
+    { value: 'isocode', type: TabbingOrderTypes.NG_SELECT },
+    { value: 'titleCode', type: TabbingOrderTypes.NG_SELECT },
+    { value: 'firstName', type: TabbingOrderTypes.FORM_FIELD },
+    { value: 'lastName', type: TabbingOrderTypes.FORM_FIELD },
+    { value: 'line1', type: TabbingOrderTypes.FORM_FIELD },
+    { value: 'line2', type: TabbingOrderTypes.FORM_FIELD },
+    { value: 'town', type: TabbingOrderTypes.FORM_FIELD },
+    { value: 'postalCode', type: TabbingOrderTypes.FORM_FIELD },
+    { value: 'isocode', type: TabbingOrderTypes.NG_SELECT },
+    { value: 'phone', type: TabbingOrderTypes.FORM_FIELD },
+    backButton,
+    continueButton,
+  ],
+  shippingAddressExisting: [
+    { value: 'Method ofPayment', type: TabbingOrderTypes.LINK },
+    { value: 'Add New Address', type: TabbingOrderTypes.BUTTON },
+    {
+      value: 'Ship to this address',
+      type: TabbingOrderTypes.LINK,
+    },
+    backButton,
+    continueButton,
+  ],
+  shippingAddressAccount: [
+    { value: 'Method ofPayment', type: TabbingOrderTypes.LINK },
+    { value: 'Cost Center', type: TabbingOrderTypes.SELECT },
+    {
+      value: 'Ship to this address',
+      type: TabbingOrderTypes.LINK,
+    },
+    backButton,
+    continueButton,
```

Comment:
using this as a placeholder to mention why this is failing sometimes in the e2e.
<img width="1440" alt="failbc" src="https://user-images.githubusercontent.com/29099466/95231333-021dd180-07d1-11eb-8ad1-f4d2ddb0c007.png">

I noticed above fails at times, and I think a fix to this is to add a check in `checkoutShippingAddressExistingTabbingOrder` before the `verifyTabbingOrder` function. 

The check would be `cy.get('cx-shipping-address .card-header').should('contain', 'Selected')` to make cypress wait for it to be ready. I noticed when it fails the selected card is missing this header, so it doesn't pick up the continue button because it's disabled. This allows it to wait for our dom to render this specific header and continue on with `verifyTabbingOrder`

cypress is a `speed demon`